SonarQube.Client.Tests/Requests/Api/V7_20/GetExclusionsRequestTests.cs

```diff
@@ -0,0 +1,171 @@
+            result.GlobalInclusions.Should().BeEmpty();
+        }
+
+        [TestMethod]
+        [Description("This is what SonarCloud returns when the setting is not defined")]
+        public async Task InvokeAsync_SomeMissingSetting_ReturnsDefinedProperties()
+        {
+            const string projectKey = "myproject";
+
+            var testSubject = CreateTestSubject(projectKey);
+
+            var handlerMock = new Mock<HttpMessageHandler>(MockBehavior.Strict);
+            var httpClient = new HttpClient(handlerMock.Object)
+            {
+                BaseAddress = new Uri(ValidBaseAddress)
+            };
+
+            var request = $"api/settings/values?component={projectKey}&keys=sonar.exclusions%2Csonar.global.exclusions%2Csonar.inclusions%2Csonar.global.inclusions";
+            var response = @"{
+	""settings"": [
+		{
+			""key"": ""sonar.global.inclusions"",
+			""values"": [
+				""**/build-wrapper-dump.json""
+			]
+		}
+	]
+}";
+
+            SetupHttpRequest(handlerMock, request, response);
+
+            var result = await testSubject.InvokeAsync(httpClient, CancellationToken.None);
+            result.Should().NotBeNull();
+
+            result.Exclusions.Should().BeEmpty();
+            result.GlobalExclusions.Should().BeEmpty();
+            result.Inclusions.Should().BeEmpty();
+            result.GlobalInclusions.Should().BeEquivalentTo("**/build-wrapper-dump.json");
+        }
+
+        [TestMethod]
+        public async Task InvokeAsync_ExistingSetting_ReturnsDefinedProperties()
+        {
+            const string projectKey = "myproject";
+
+            var testSubject = CreateTestSubject(projectKey);
+
+            var handlerMock = new Mock<HttpMessageHandler>(MockBehavior.Strict);
+            var httpClient = new HttpClient(handlerMock.Object)
+            {
+                BaseAddress = new Uri(ValidBaseAddress)
+            };
+
+            var request = $"api/settings/values?component={projectKey}&keys=sonar.exclusions%2Csonar.global.exclusions%2Csonar.inclusions%2Csonar.global.inclusions";
+            var response = @"{
+	""settings"": [
+		{
+			""key"": ""sonar.exclusions"",
+			""values"": [
+				""**/value1"",
+				""value2"",
+				""some/value/3"",
+			]
+		},
+		{
+			""key"": ""sonar.global.exclusions"",
+			""values"": [
+				""some/value/4"",
+			]
+		},
+		{
+			""key"": ""sonar.inclusions"",
+			""values"": []
+		},
+		{
+			""key"": ""sonar.global.inclusions"",
+			""values"": [
+				""**/value/5/6/7"",
+				""value/**/8"",
+			]
+		}
+	]
+}";
+
+            SetupHttpRequest(handlerMock, request, response);
+
+            var result = await testSubject.InvokeAsync(httpClient, CancellationToken.None);
+            result.Should().NotBeNull();
+
+            result.Exclusions.Should().BeEquivalentTo("**/value1", "value2", "some/value/3");
+            result.GlobalExclusions.Should().BeEquivalentTo("some/value/4");
+            result.Inclusions.Should().BeEmpty();
```

Comment:
There isn't a test that checks `inclusions` correctly returns a value, is there? i.e. the `InclusionsKey` constant could be changed and all of the tests would still pass.